### PR TITLE
feat: created openapi spec

### LINKED ISSUE #2: Design API spec for the service
We would like to have an OpenAPI spec for the service.

### PR DESCRIPTION
**Pull request by CptSchnitz**
closes #2

### CODE REVIEW

**Comment by galta95** [comment]
pleaseeeeeeee change the repo name to oms-sync-tracker

**Comment by CptSchnitz** [comment]
image

**Comment by galta95** [comment]
Lets add bulk insert to entities, files.

**Review by galta95 — CHANGES_REQUESTED** [review]

File: openapi3.yaml [hunk 1]
```diff
@@ -1,64 +1,354 @@
...
+          type: string
+          format: date-time
+        endDate:
+          type: string
+          format: date-time
+        layerId:
+          $ref: '#/components/schemas/layerId'
+        isFull:
+          type: boolean
+        filesTotal:
```
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply openapi3.yaml]: done

File: openapi3.yaml [hunk 2]
```diff
@@ -1,64 +1,354 @@
...
+    file:
+      type: object
+      properties:
+        fileId:
+          $ref: '#/components/schemas/uniqueId'
+        syncId:
+          allOf:
+            - $ref: '#/components/schemas/uniqueId'
+            - readOnly: true
+        entitiesTotal:
```
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply openapi3.yaml]: done

File: openapi3.yaml [hunk 3]
```diff
@@ -1,64 +1,354 @@
...
+        filesTotal:
+          type: integer
+    sync:
+      allOf:
+        - type: object
+          properties:
+            id:
+              $ref: '#/components/schemas/uniqueId'
+        - $ref: '#/components/schemas/syncWithoutId'
+    file:
```
[the remark this answers is not in the document]
Comment (CptSchnitz) [reply openapi3.yaml:378]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**